movement/movement.h

```diff
@@ -109,13 +110,16 @@ typedef enum {
     EVENT_TIMEOUT,              // Your watch face has been inactive for a while. You may want to resign, depending on your watch face's intended use case.
     EVENT_LIGHT_BUTTON_DOWN,    // The light button has been pressed, but not yet released.
     EVENT_LIGHT_BUTTON_UP,      // The light button was pressed and released.
-    EVENT_LIGHT_LONG_PRESS,     // The light button was held for >2 seconds, and released.
+    EVENT_LIGHT_LONG_PRESS,     // The light button was held for >2 seconds, but not yet released.
```

Comment:
Comment should say 1/2 second, perhaps? (I know, this issue already existed)

Reply:
You are right, the >2 seconds comment seems to be outdated. Will be corrected in next commit.